config/proco.yml

```diff
@@ -0,0 +1,17 @@
+# PURL configuration for http://purl.obolibrary.org/obo/proco
+
+idspace: PROCO
+base_url: /obo/proco
+
+products:
+- proco.owl: https://raw.githubusercontent.com/proco-ontology/PROCO/v2022-04-11/proco.owl 
```

Comment:
You should not use a version IRI as the redirect for a main product.. Unless you want to use github releases, you should probably use https://raw.githubusercontent.com/proco-ontology/PROCO/main/proco.owl here.